tailwind.config.ts

```diff
@@ -10,6 +10,13 @@ const config: Config = {
     theme: {
     	extend: {
     		colors: {
+				primarydarkbg: '#020010',
+				secondarydarkbg: '#0c1117',
+				authcardbg: '#09090B',
+				authborder: '#27272A',
+				bordersworkspace: '#3D4249',
```

Comment:
```suggestion
				bordersworkspace: '#3D4249',
```

can we come up with better name for this?

Reply:
Well I don't think we have to bother much about auth color since we'd grab the final component from ShadCN, and borderworkspace , workspacebr?